sc2reader/factories/plugins/replay.py

```diff
@@ -100,21 +100,30 @@ def APMTracker(replay):
     above actions divided by the number of seconds played by the player (not
     necessarily the whole game) multiplied by 60.
     """
+
+    speed_multiplier = 1
+    if replay.expansion == 'LotV':
+        speed_multiplier = 1.4
+
+    game_seconds_per_second = 1.4
+    if replay.expansion == 'LotV':
+        game_seconds_per_second = 1.4
```

Comment:
Yeah, you're right. The test was affected, so it is impacted by the change. I tried to fix it but now it just make even less sense to me. I think I need a second iteration on this to understand if we actually can separate APM calculation and APM visualization in two independent tasks.

Afaiu, APM is actions-per-game-minute, so it is affected by the LotV re-definition of a game-minute. A player has one fixed amount of actions in a replay, so shortening down the game-length of a replay increases the APM - both avg APM and the values of the points in the series in the APM chart. As you noted somewhere else, the 22 observations need to be fitted into the 15 timeslots.